### DISCUSSION TITLE
Welcome to SkyeTracker Discussions!

### DISCUSSION THREAD

**Discussion by graham22** [ClassicDIY/SkyeTracker#30] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by petez69** [comment]
Hi Graham

Based in the middle of Australia where we get tons of sun :-)  Town is Alice Springs which is equidistant to every beach in Australia :-)  Had been looking to build a Sun Tracker, why because I can.  Your project looks spot on, exactly what I'm after and it has MQTT integrastion, thats great :-)

One question I do have, what size actuators did you use ?  Ive never used any but they look straight forward.  Clearly issues of loading etc need to be considered but I'm just trying to get my head around how close to the horizon should I try and get to.....The mechanical construction itself looks fairly straight forward

Any pointers would be appreciated....Pete

**Reply by graham22** [reply]
This might be the first time it's used in the southern hemisphere, it should work although you might have to reverse the horizontal actuators polarity so that fully extended points east or build the frame with the H actuator on the other side of the shaft.
I just used these ([cheap actuators](https://www.princessauto.com/en/12v-dc-135-lb-linear-actuators/product/PA1000001114?skuId=8489098) for a 2 panel setup. Using the app, you can configure the speed and limits of the actuators.

Graham.
